src/System.Buffers/src/project.json

```diff
@@ -1,11 +1,16 @@
 {
+  "compilationOptions": {
+    "allowUnsafe": true
+  },
   "dependencies": {
-    "System.Diagnostics.Debug": "4.0.0", 
+    "System.Diagnostics.Debug": "4.0.0",
+    "System.Diagnostics.Tools": "4.0.1-beta-23409",
     "System.Reflection.Primitives": "4.0.0",
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.Extensions": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
+    "System.Slices": "0.1.0-d103015-1",
```

Comment:
That would totally makes sense if those projects were in different repositories but this will just increase friction. However, if that's the goal, it up to you guys. I just want to call out that it is possible to reference the project (if that was desirable), in place without changing the structure.